tracker/view/add.py

```diff
@@ -38,6 +41,15 @@ def add_cve():
 
     cve = db.get(CVE, id=form.cve.data)
     if cve is not None:
+        advisories = (db.session.query(Advisory)
+                      .join(CVEGroupEntry, CVEGroupEntry.cve_id == cve.id)
+                      .join(CVEGroup, CVEGroupEntry.group)
+                      .join(CVEGroupPackage, CVEGroup.packages)
+                      .filter(Advisory.group_package_id == CVEGroupPackage.id)).all()
+        if not user_can_edit_issue(advisories):
+             flash(ERROR_ISSUE_EXISTS.format(cve.id), 'error')
+             return redirect('/{}'.format(cve.id))
```

Comment:
i think we should have matching http status codes like we do in edit.py and just return forbidden()

Reply:
Makes sense, I didn't know that the flash would be present on an error page as well, but that works nicely :)